server/models/Profile.js

```diff
@@ -0,0 +1,40 @@
+import mongoose from 'mongoose'
+
+const profileSchema = mongoose.Schema({
+	profile_photo: {
```

Comment:
The standard naming convention for field names is camelCase, please change it accordingly

Ditto throughout!